### ISSUE TITLE
KeyboardDisplayRequiresUserAction workaround

### ISSUE THREAD

**Issue by clauderic** [Telerik-Verified-Plugins/WKWebView#184]
Hey @EddyVerbruggen, I know as per https://github.com/Telerik-Verified-Plugins/WKWebView/issues/46 that supporting `keyboardDisplayRequiresUserAction` out of the box for WKWebView is not possible. 

However, someone on the JIRA bug report for this appears to have found a workaround, but I'm way out of my depth trying to apply it. Think you could take a look? https://bugs.webkit.org/show_bug.cgi?id=142757#c3

His workaround:
```
I've managed to work-around this on at least iOS 9 by swizzling
_startAssistingNode:userIsInteracting:blurPreviousNode:userObject
on  WKContentView and overriding the userIsInteracting value:

Pseudo code:

    swizzle_intercept("WKContentView", "_startAssistingNode:userIsInteracting:blurPreviousNode:userObject:", &hackAssist);

    void hackAssist (id self, SEL _cmd, void* arg0, BOOL arg1, BOOL arg2, id arg3) {
        ((void (*)(id,SEL,void*,BOOL,BOOL,id))swizzle_interceptee(hackAssist))(self, _cmd, arg0, TRUE, arg2, arg3);
    }

Use it to your advantage while the API doesn't change ;). Cheers!
```

**Comment by EddyVerbruggen** [comment]
Hey man, you got me on the right track! Support has just landed in 0.6.5, thanks on behalf of all users of this plugin :+1:

**Comment by clauderic** [comment]
Holy crap that was fast. I can't even begin to find the words to thank you man, you just made my day :raised_hands: 

Thanks for the awesome work you put in this plugin man, it's truly appreciated :+1:

**Comment by ccorcos** [comment]
Hey there, this doesn't work anymore in the latest iOS 11.3 :( Any ideas?

**Comment by SevenSystems** [comment]
We've also been hit by this in our app Xequence, and it's amazing how Apple keeps breaking things for the sake of "usability", breaking that same usability in the process. I'll keep experimenting if I can get this sorted but it ain't looking good.

**Comment by vtourraine** [comment]
Fix for iOS 11.3: https://stackoverflow.com/a/48623286/135712
